### ISSUE TITLE
Three.js v105+ and @johh/three-effectcomposer

### ISSUE THREAD

**Issue by codesignist** [akella/webgl-mouseover-effects#6]
This package has been deprecated.
https://www.npmjs.com/package/@johh/three-effectcomposer

Do you have plan to update this example to Three.js v105?

**Comment by akella** [comment]
i dont think this one would be updated, as it is an illustration from the article. Just use native three.js effectcomposer, it will work the same way
